docs/dnspolicy/dns-provider.md

```diff
@@ -7,62 +7,92 @@ In order to be able to interact with supported DNS providers, Kuadrant needs a c
 
 Kuadrant Supports the following DNS providers currently
 
-- AWS route 53 (aws)
-- Google DNS (gcp)
+- AWS Route 53 (AWS)
+- Google Cloud DNS (GCP)
 
+### AWS Route 53 Provider
 
+Kuadant expects a `Secret` with a credential. Below is an example for AWS Route 53. It is important to set the secret type to `aws`:
 
-## Configuring an AWS Route 53 provider
 
-Kuadant expects a secret with a credential. Below is an example for AWS Route 53. It is important to set the secret type to `aws`
-
-```
-apiVersion: v1
-data:
-  AWS_ACCESS_KEY_ID: XXXXX
-  AWS_REGION: XXXXX
-  AWS_SECRET_ACCESS_KEY: XXXXX
-kind: Secret
-metadata:
-  name: aws-credentials
-  namespace: multicluster-gateway-controller-system
-type: kuadrant.io/aws
+```bash
+kubectl create secret generic my-aws-credentials \
+  --namespace=multicluster-gateway-controller-system \
+  --type=kuadrant.io/aws \
+  --from-literal=AWS_ACCESS_KEY_ID=XXXX \
+  --from-literal=AWS_REGION=eu-west-1 \
+  --from-literal=AWS_SECRET_ACCESS_KEY=XXX
 ```
 
+| Key                      | Example Value           | Description                                           |
+|--------------------------|-------------------------|-------------------------------------------------------|
+| `AWS_REGION`             | `eu-west-1`             | AWS Region                                            |
+| `AWS_ACCESS_KEY_ID`      | `XXXX`                  | AWS Access Key ID (see note on permissions below)     |
+| `AWS_SECRET_ACCESS_KEY`  | `XXXX`                  | AWS Secret Access Key                                 |
+
+#### AWS IAM Permissions Required 
+We have tested using the available policy `AmazonRoute53FullAccess` however it should also be possible to restrict the credential down to a particular zone. More info can be found in the AWS docs:
 
-## IAM permissions required 
-We have tested using the available policy `AmazonRoute53FullAccess` however it should also be possible to restrict the credential down to a particular zone. More info can be found in the AWS docs 
 https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/access-control-managing-permissions.html
 
-## Configuring a Google DNS provider
+### Google Cloud DNS Provider
 
-Kuadant expects a secret with a credential. Below is an example for Google DNS. It is important to set the secret type to `gcp`
+Kuadant expects a secret with a credential. Below is an example for Google DNS. It is important to set the secret type to `gcp`:
 
+```bash
+kubectl create secret generic my-test-gcp-credentials \
+  --namespace=multicluster-gateway-controller-system \
+  --type=kuadrant.io/gcp \
+  --from-literal=PROJECT_ID=xxx \
+  --from-file=GOOGLE=$HOME/.config/gcloud/application_default_credentials.json
 ```
```

Comment:
:+1: this is how we should be telling people how to create secrets for providers. If you use gcloud, you will be familiar with how it works, and so understand what the credentials.json file is.